gcloud/storage/blob.py

```diff
@@ -91,8 +90,6 @@ def path_helper(bucket_path, blob_name):
     @property
     def acl(self):
         """Create our ACL on demand."""
-        if self._acl is None:
-            self._acl = ObjectACL(self)
```

Comment:
Now that it is no longer lazily computed, are we retaining it as a property (rather than renaming the attribute from `_acl` to `acl`) to prevent assignment?